### PR TITLE
Use low cardinality span names in Cassandra instrumentation

### PR DESCRIPTION
**Pull request by mateuszrzeszutek**
Part of https://github.com/open-telemetry/opentelemetry-java-instrumentation/issues/2128

### CODE REVIEW

**Review by iNikem — APPROVED** [review]

File: instrumentation/cassandra/cassandra-3.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/cassandra/v3_0/CassandraDatabaseClientTracer.java [hunk 1]
```diff
@@ -34,22 +35,27 @@ protected String getInstrumentationName() {
 
   @Override
   protected SqlStatementInfo sanitizeStatement(String statement) {
-    return SqlStatementSanitizer.sanitize(statement);
+    return SqlStatementSanitizer.sanitize(statement).mapTable(this::stripKeyspace);
```
Comment (trask) [inline instrumentation/cassandra/cassandra-3.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/cassandra/v3_0/CassandraDatabaseClientTracer.java:38]: why strip keyspace? I think that's equivalent of schema, which should be included in `db.sql.table`:

https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/database.md#call-level-attributes-for-specific-technologies

Comment (mateuszrzeszutek) [reply instrumentation/cassandra/cassandra-3.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/cassandra/v3_0/CassandraDatabaseClientTracer.java:38]: It was added here: https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/2177

@FrankSpitulski can you explain why we're stripping the keyspace?

Comment (FrankSpitulski) [reply instrumentation/cassandra/cassandra-3.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/cassandra/v3_0/CassandraDatabaseClientTracer.java:38]: We already have the keyspace as a separate tag and stripping it makes it more consistent for searching. Without stripping you’d have to search for key.table and table if you wanted queries on a table.

Comment (trask) [reply instrumentation/cassandra/cassandra-3.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/cassandra/v3_0/CassandraDatabaseClientTracer.java:38]: oh I missed that this is capturing `db.cassandra.table` and not `db.sql.table`

though the spec says the same thing for `db.cassandra.table`:

> The name of the primary table that the operation is acting upon, including the schema name (if applicable)

https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/database.md#cassandra

I created issue #2476 to discuss/resolve the difference between our impl and the spec, so we can ignore this issue for this PR 😄

**Review by trask — APPROVED** [review]

**Pull request merged**